index.mako

```diff
@@ -79,9 +79,17 @@ pre-commit
 
 Using [homebrew](https://brew.sh):
 
-```
+```bash
 brew install pre-commit
 ```
+
+Using [conda](https://conda.io) (via [conda-forge](https://conda-forge.org)):
+
+```bash
+conda config --add channels conda-forge
+conda install pre_commit
```

Comment:
I think it is better to advertise:

```
conda install -c conda-forge pre_commit
```

This is what's suggested [here](https://anaconda.org/conda-forge/pre_commit), also the former would change the global configuration which is not very nice.

Reply:
nice! pretty clear that

![](https://i.imgur.com/AfgkjCf.jpg)